torch/lib/c10d/ProcessGroupNCCL.cpp

```diff
@@ -271,19 +286,8 @@ ProcessGroupNCCL::ProcessGroupNCCL(
       ncclCommCounter_(0),
       terminateWatchdog_(false),
       opTimeout_(opTimeout) {
```

Comment:
We can alternately set `blockingWait_` from the `ProcessGroupNCCL` ctor initializer list and handle all exception cases in the helper - would that be better style-wise?

Reply:
Current solution looks good to me. Initialization list tends to do trivial initialization.